src/Page.js

```diff
@@ -344,6 +345,28 @@ Page.prototype.concatenateHeadingsAndKeywords = function () {
   });
 };
 
+/**
+ * Filters out elements on the page based on config tags
+ * @param tags to filter
+ * @param content of the page
+ */
+Page.prototype.filterTags = function (tags, content) {
+  if (tags === undefined) {
+    return content;
+  }
```

Comment:
I wonder if we should warn the user if the tags field that they used is not a list (or help the user convert the single value to a list):

```
<frontmatter>
  tags: "language--C#"
</frontmatter>
```

```
error: TypeError: tags.forEach is not a function
warning: Error: Error while generating C:\Users\Tan Wang Leng\Desktop\New folder\index.md
```

Reply:
Sure, I can wrap it in an array if it isn't one

Reply:
I've tested this manually and it works - not sure how to add an automated test for it since the tags get overwritten.

Reply:
> I've tested this manually and it works - not sure how to add an automated test for it since the tags get overwritten.

I guess we are hitting the limit of having only one `test_site`. Having multiple `test_site` would allow us to test different combinations.

I may just open a new issue for it, no need to worry about it for now.